src/test/java/io/vertx/ext/mail/MailConfigTest.java

```diff
@@ -26,8 +26,9 @@
   @Test
   public void toJsonTest() {
     MailConfig mailConfig = new MailConfig();
-    assertEquals("{\"hostname\":\"localhost\",\"port\":25,\"starttls\":\"OPTIONAL\",\"login\":\"NONE\",\"maxPoolSize\":10}", mailConfig
```

Comment:
TBH, I'm not in love with this style of testing (since the exact format/ordering of the JSON document is presumably not part of the API's contract).  However, I'm trying to make the least change here.